docs/guide/migrating-to-version-9.md

```diff
@@ -30,13 +30,36 @@ Kongponents styles are no longer designed to be utilized standalone, separately
 
 ### KBadge
 
+#### Props
+
+* `appearance` prop no longer accepts `default` and `custom` values (defaults to `info` now)
+* `isBordered` prop has been removed
+* `shape` prop has been removed (now KBadge only comes in one what used to be `rectangular` shape)
+* `borderColor` prop has been removed
+* `backgroundColor` prop has beed removed
+* `color` prop has been removed
+* `dismissable` prop has been removed. You can now pass a clickable close icon (remember to add `role="button"` and appropriate `tabindex` attributes) through the `icon` slot and handle KBadge showing/hiding from your host app
```

Comment:
in the two places in this PR where you mention `tabindex` should we mention likely using a tabindex of `0` so people don't go crazy adding higher values that aren't needed?

Reply:
What if it's disabled? Then `tabindex` should be -1 -- basically instead of mentioning that I thought phrasing "appropriate `tabindex`" should do.

Reply:
if it's disabled, wouldn't you just not add a tabindex at all?

Reply:
It could be conditional but whatever, I think it's pretty narrow use-case - updated this block to say `tabindex="0"`